README.md

```diff
@@ -1,51 +1,42 @@
 [![Build Status](https://travis-ci.com/NYPL/discovery-api.svg?branch=main)](https://travis-ci.com/NYPL/discovery-api)
 
-# Discovery API
+# Discovery API (aka Research Catalog API)
 
 This is the API providing most of bibliographic data to the [NYPL Research Catalog front-end](https://github.com/NYPL/discovery-front-end). Check the [current swagger](https://github.com/NYPL/discovery-api/blob/main/swagger.v1.1.x.json) for the machine readable api contract.
 
 ## Installing & Running Locally
 
-Use [nvm](https://github.com/creationix/nvm) to set your Node version:
+For local development, it's easiest to just use local node binaries:
 
 ```
-nvm use
+nvm use; npm i
+nvm use; ENV=qa npm start
 ```
 
-Install dependencies:
+Note that when developing locally, you may need to [add your IP to the access control policy of the relevant ES domain](https://github.com/NYPL/aws/blob/b5c0af0ec8357af9a645d8b47a5dbb0090966071/common/elasticsearch.md#2-make-the-domain-public-restrict-by-ip).
```

Comment:
Can this include a note about what the error looks like when you don't have permissions  eg `{"Message":"User: anonymous is not authorized to perform: es:ESHttpGet"}`? I remember being confused by this a lot when I started working with this repo.

Reply:
I added more detail to this documentation - and better app handling so that the specific error is clearer in app logs